packages/bsky/src/api/app/bsky/feed/searchPosts.ts

```diff
@@ -50,6 +50,16 @@ const skeleton = async (inputs: SkeletonFnInput<Context, Params>) => {
         q: params.q,
         cursor: params.cursor,
         limit: params.limit,
+        author: params.author,
+        domain: params.domain,
+        lang: params.lang,
+        mentions: params.mentions,
+        since: params.since,
+        sort: params.sort,
+        tag: params.tag,
+        until: params.until,
+        url: params.url,
+        viewer: params.hydrateCtx.viewer ?? undefined,
```

Comment:
to to clarify (I had forgotten for a sec), the reason we pass this to the backend here is to support `from:me` search syntax (aka, "search through my own posts").